src/app/core/services/label.service.ts

```diff
@@ -13,6 +13,11 @@ const COLOR_DARKNESS_THRESHOLD = 0.184;
 const COLOR_BLACK = '000000'; // Dark color for text with light background
 const COLOR_WHITE = 'ffffff'; // Light color for text with dark background
 
+export type simplifiedLabel = {
+  name: string;
+  color: string;
+};
+
```

Comment:
For another issue/PR: This would be good to resolve why we need Label vs SimplifiedLabel, it seems redundant to have both